test/unit/org/apache/cassandra/metrics/BatchMetricsTest.java

```diff
@@ -145,21 +145,33 @@ private void assertMetrics(BatchStatement.Type batchTypeTested, int[] rounds, in
         for (int ix = 0; ix < rounds.length; ix++)
         {
             long partitionsPerLoggedBatchCountPre = metrics.partitionsPerLoggedBatch.getCount();
+            long columnsPerLoggedBatchCountPre = metrics.columnsPerLoggedBatch.getCount();
```

Comment:
I suppose adding these definitions in one consistent block would be less conflict-prone on rebase.
That said if anything changes in the original definitions perhaps we *do* want to get a conflict?

I leave it up to you to decide.